src/test/java/processors/StreamProcessorTest.java

```diff
@@ -1,30 +1,35 @@
 package processors;
 
 import java.io.BufferedReader;
-import java.io.InputStream;
+import java.util.Arrays;
 import java.util.function.Consumer;
+import java.util.regex.Pattern;
 
-import junit.framework.Assert;
-
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.Mockito;
 import org.mockito.runners.MockitoJUnitRunner;
 
+import junit.framework.Assert;
+
 @RunWith(MockitoJUnitRunner.class)
 public class StreamProcessorTest {
 	@Mock
 	BufferedReader bufferedInputMock;
 	@Mock
-	InputStream input;
-	@Mock
 	Consumer<String> consumer;
+	@Mock
+	Consumer<String> consumer2;
 
-	@InjectMocks
 	private StreamProcessor testee;
 
+	@Before
+	public void setup() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [You may have misspelled a JUnit framework method (setUp or tearDown)](https://www.codacy.com/app/joegotgit/java-log-file-filter-tool/file/2552765350/issues/source?bid=3266573&fileBranchId=3290347#l29)